lib/js/dom.nim

```diff
@@ -1297,3 +1307,11 @@ proc offsetHeight*(e: Node): int {.importcpp: "#.offsetHeight", nodecl.}
 proc offsetWidth*(e: Node): int {.importcpp: "#.offsetWidth", nodecl.}
 proc offsetTop*(e: Node): int {.importcpp: "#.offsetTop", nodecl.}
 proc offsetLeft*(e: Node): int {.importcpp: "#.offsetLeft", nodecl.}
+
+
+since (1, 3):
+  func newDOMParser*(): DOMParser {.importcpp: "(new DOMParser()​​)".}
```

Comment:
this doesn't work, you must've pasted from a webpage and introduced an invisible symbol. you should make sure examples work

Reply:
This should be ``.importjs: "new DOMParser()"``

Reply:
https://github.com/nim-lang/Nim/issues/13891#issue-594824308